src/modules/components/proposal/proposalActions/proposalActions/proposalActions.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/react';
 
+import { generateProposalActionUpdateMetadata } from '../actions/generators';
 import { generateProposalActionWithdrawToken, generateToken } from '../actions/generators/proposalActionWithdrawToken';
```

Comment:
As you're adding the barrel file, we can import the generators with the same import statement
```suggestion
import { generateProposalActionUpdateMetadata, generateProposalActionWithdrawToken, generateToken } from '../actions/generators';
```